### PR TITLE
Document <LatitudeEmbed />

### PR DESCRIPTION
**Pull request by andresgutgon**
## What?
Document react embedding

### CODE REVIEW

File: docs/guides/embed/react-embed.mdx [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  1. Enables your React app to notify the Latitude app when a parameter is modified within your React app.
+  2. Allows for the reverse, in which the Latitude app communicates parameter changes back to your React app.
+  3. Gives you the ability to execute all or selected queries directly from your React app.
+  4. Permits the dispatch of custom events from your React app to the Latitude view.
+</Note>
+
+We have an [example application](https://github.com/latitude-dev/latitude/tree/main/examples/sample-react). You can check the source code if you get lost in the process.
+In [Embedding/index.tsx](https://github.com/latitude-dev/latitude/blob/main/examples/sample-react/src/examples/Embedding/index.tsx) you can see all the options you can pass to the `<LatitudeEmbed />` component.
+
+Try to change in this example the field in `YOUR REACT APP` and notice how the Latitude app changes. Now do the same but inside `LATITUDE IFRAME` and notice how the React app changes. Click `Run` button, this will run all the queries in the example.
```
Comment (cesr) [inline docs/guides/embed/react-embed.mdx:22]: This part is not super clear

File: docs/guides/embed/react-embed.mdx [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+And lastly `Pick your character` and see how Latitude app shows your Star Wars character. May the force be with you!
+
+<iframe class="rounded-xl" src="https://latitude-embed.vercel.app/embedding" width="100%" height="800"></iframe>
+
+But let's go through the options you can pass to the `<LatitudeEmbed />` component.
+Let's walk through the process of embedding a Latitude data app within a React application using the `<LatitudeEmbed />` component.
```
Comment (cesr) [inline docs/guides/embed/react-embed.mdx:29]: same thing twice

File: docs/guides/embed/react-embed.mdx [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+ReactDOM.createRoot(document.getElementById('root')!).render(
+  <React.StrictMode>
+    {/* Your app goes here */}
+    <App />
+  </React.StrictMode>,
+)
+```
+
+And lastly use `<LatitudeEmbed />` component to embed the Latitude app.
```
Comment (cesr) [inline docs/guides/embed/react-embed.mdx:57]:
```suggestion
And lastly use the `<LatitudeEmbed />` component to embed the Latitude app.
```

File: docs/guides/embed/react-embed.mdx [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+        // Update the end year in your React app
+        // when the end year in the Latitude app changes
+        setEndYear(newEndYear)
+      }}
+    />
+  )
+}
+```
+
+### Execute queries
```
Comment (cesr) [inline docs/guides/embed/react-embed.mdx:140]:
```suggestion
### Run queries
```

File: docs/guides/embed/react-embed.mdx [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+You can execute all or selected queries directly from your React app.
```
Comment (cesr) [inline docs/guides/embed/react-embed.mdx:141]:
```suggestion
You can run all queries or specific ones directly from your React app.
```

File: docs/guides/embed/react-embed.mdx (hunk 1 above)
Comment (andresgutgon) [reply docs/guides/embed/react-embed.mdx:22]: I changed it anyway users will have the context of the example app
image

**Pull request merged**